runtime/compiler/optimizer/JProfilingValue.hpp

```diff
@@ -45,20 +45,18 @@ class TR_JProfilingValue : public TR::Optimization
    virtual const char *optDetailString() const throw();
 
    void lowerCalls();
-   void removeRedundantProfilingValueCalls();
-   void addProfiling(TR::Node *address, TR::TreeTop *tt);
-   void addVFTProfiling(TR::Node *address, TR::TreeTop *tt, bool addNullCheck);
-   void performOnNode(TR::Node *node, TR::TreeTop *tt, TR::NodeChecklist *checklist);
-
+   void cleanUpAndAddProfilingCandidates();
+   void performOnNode(TR::Node *node, TR::TreeTop *tt, TR_BitVector *alreadyProfiledValues, TR::NodeChecklist *checklist);
```

Comment:
Could you please add doxygen for the various functions you have added/updated to make it easier to follow what this thing is doing?

Reply:
I have added doxygen comments for these newly added/updated functions in header file https://github.com/eclipse/openj9/pull/6585/commits/4babc9149b99cc98d9703d5340c8c21f81c918da